userdocs/profiles.dev/docs/tutorial-basics/install-a-profile.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 3
+---
+
+# Install a profile
+
+:::info
+
+This stage of the tutorial assumes you have prepared your environment correctly.
+
+Please refer back to the [set up docs](/docs/tutorial-basics/setup) if not.
+:::
+
+To install a profile, we use `pctl`.
+
+First, take care to ensure that you are in your GitOps repo directory on your local
+filesystem. Your GitOps repo is the one you synced to Flux in your cluster in the
+[environment setup](/docs/tutorial-basics/setup#a-github-repo-synced-to-flux) section of this tutorial.
+
+```bash
+git clone <URL to your cluster gitops repo> # if this does not exist locally yet
+cd <path to repository dir>
+```
+
+Once in the correct directory, we can run `pctl` to install a profile.
+_(A breakdown of each flag is provided below.)_
+
+With the following command `pctl` will generate a set of manifests for each profile artifact,
+commit those manifests to a branch in your GitOps repo, push that branch and open a
+PR on your repo to merge the changes.
+
+```bash
+pctl install \
+  --profile-url <URL of profile to install> \
+  --profile-path=. \
+  --create-pr \
+  --pr-repo <gitops repo username or orgname>/<gitops repo name> \
+  --pr-branch install-simple-profile
+```
```

Comment:
missing `--git-repository`. although in your example I guess its not technically needed, since theres no local artifacts. but might be worth getting the user used to it as a concept? idk

Reply:
not in the basics tutorial